client/state/selectors/is-account-recovery-reset-options-ready.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Return a boolean value indicating whether the account recovery reset options are fetched and ready to be used.
+ *
+ * @param  {Object} state Global state tree
+ * @return {Boolean}      If the reset options are ready.
+ */
+export default ( state ) => {
+	const resetOptions = state.accountRecovery.reset.options;
+	return null == resetOptions.error && 0 < resetOptions.items.length;
```

Comment:
Got it :)

The style guide [doesn't enforce](https://github.com/Automattic/wp-calypso/blob/afe8a0f449d233a862d41b3f483f1b2835c838b7/docs/coding-guidelines/javascript.md#yoda-conditions) Yoda conditions, it's personal preference and fine to leave it as is!